core/resource/service.go

```diff
@@ -467,6 +479,9 @@ func (s Service) listUserResources(ctx context.Context, resourceType string, use
 		actMap[action] = true
 		resources, err := s.relationService.LookupResources(ctx, resourceType, action, userNamespace, user.ID)
 		if err != nil {
+			if strings.Contains(err.Error(), "not found under definition") {
```

Comment:
can we check using this https://github.com/authzed/spicedb/blob/c5a5affb9e735442266eedd63b5dc97344199dc1/internal/dispatch/graph/errors.go#L41 instead of checking the plain string here?

Reply:
This could've been better idea, but I'm not really sure if we can use it since it's in internal package

Reply:
lets put comment on this line and put the reference to the code base?

Reply:
sure